### PR TITLE
doc(flatglass): Add Showcase specification for Flatglass @ glasstec

### PR DESCRIPTION
**Pull request by karstenroethig**

### CODE REVIEW

File: Specs/Flatglass.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# umati showcase information model for Flat Glass Processing
+
+In the following the umati showcase describes the additional information on top of the [OPC 40301 UA for Flat Glass Processing](https://www.vdma.org/viewer/-/v2article/render/15410388) companion specification.
+
+Please refer to the provided xml-Nodesets or UaModeler-project for exemplary implementation.
```
Comment (GoetzGoerisch) [inline Specs/Flatglass.md:5]: @karstenroethig @Kantiran91 do you intend to provide also a UaModeler project?

File: Specs/Flatglass.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+The following describes the umati showcase information model for Flat Glass Processing.
+
+For participants that are familiar with the information model, we have included comments and descriptions about changes between the release candidate information model and the showcase information model.
+
+All participants not familiar with the release candidate information model do not need to review these changes in detail but rather follow the described model and refer to the provided xml Nodeset files and UA modeler project.
+
+## NodeSets for the showcase
+
+[Normative NodeSet hosted by the OPC Foundation](https://github.com/OPCFoundation/UA-Nodeset/tree/v1.04/Glass/Flat)
```
Comment (GoetzGoerisch) [inline Specs/Flatglass.md:15]: Change link: https://github.com/OPCFoundation/UA-Nodeset/tree/latest/Glass/Flat

File: Specs/Flatglass.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+| 0:HasInterface | ObjectType | IMachineTagNameplateType |
+| Applied from IMachineVendorNameplateType |     |     |     |     |     |
+| 0:HasProperty | Variable | 2:ProductInstanceUri | 0:String | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | 2:Manufacturer | 0:LocalizedText | 0:PropertyType | M, RO |
+| 0:HasProperty | Variable | 2:ManufacturerUri | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | **2:Model** | 0:LocalizedText | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | **2:ProductCode** | 0:String | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | 2:HardwareRevision | 0:String | 0:PropertyType | O, RO |
+| 0:HasProperty | Variable | **2:SoftwareRevision** | 0:String | 0:PropertyType | **M**, RO |
+| 0:HasProperty | Variable | 2:DeviceClass | 0:String | 0:PropertyType | **M**, RO |
```
Comment (GoetzGoerisch) [inline Specs/Flatglass.md:48]: Strong **DeviceClass**

**Review by GoetzGoerisch — CHANGES_REQUESTED** [review]
Great! Please resolve the comments inline and the Linters errors.
As a favor, please prepare 1.07.0 release.

Good to merge when green.

File: Specs/Flatglass.md (hunk 1 above)
Comment (karstenroethig) [reply Specs/Flatglass.md]: Copy-paste error. I have deleted it.

File: Specs/Flatglass.md (hunk 3 above)
Comment (karstenroethig) [reply Specs/Flatglass.md]: Changed it to bold.

File: Specs/Flatglass.md (hunk 2 above)
Comment (karstenroethig) [reply Specs/Flatglass.md]: Changed it.

**Comment by karstenroethig** [comment]
Honestly, I don't get these lint errors
- line length no more than 400? But it looks good on the website. Should I just insert a line break somewhere?
- Changing "Node" to "Node.js" would just be wrong
- Changing "website" to "site" changes the context in my opinion
- When I change "html" to "HTML", the link doesn't work anymore

**Pull request merged**